src/content/docs/ai-observability/advanced-customization/drop-sensitive-data.mdx

```diff
@@ -0,0 +1,306 @@
+  ```sql
+  SELECT content FROM LlmChatCompletionMessage WHERE content RLIKE ^([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})$
+  ```
+1. Repeat the above steps to create drop filters for the remaining events and column pairs. 
+
+## Your data compliance responsibilities [#responsibilities]
+
+New Relic can't guarantee that this functionality completely resolves your data disclosure concerns, nor can we provide support for building out your NRQL queries. We recommend that you:
+
+* Review your drop rules and confirm they're accurately identifying and discarding the data you want dropped.
+* Check that your drop rules are still dropping sensitive information after you've created them. 
+
+While drop rules ensure that personal information about your end users aren't stored in NRDB, creating the rules themselves imply the kinds of data you maintain, including the format of your data or systems. This is important when determining control permissions for certain users in your org, as certain permissions let users view and edit all information in the rules you create.
+
+## How drop filters work [#drop-rules-work]
+
+A drop filter directs the agent to drop information if it meets a set of conditions. A drop filter contains three parts:
+
+* Events: A stored record from an interaction within your system.
+* Attributes, or columns: A key-value pair attached to data objects. 
+* Regex: A string of characters and operators that corresponds to kinds of information. 
+
+### Events and attributes 
+
+In a typical AI interaction, a prompt or request undergoes certain processes (like embedding) that are recorded as discrete events. For example, let's say a customer requests a street address on file. The prompt is processed by the model, which pulls additional context through various services and databases. Your AI assistant then returns with a response that contains the requested information. 
+
+Below are the different events that occur in a system in an interaction like this: 
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Event
+            </th>
+            <th>
+                Attribute, or column 
+            </th>
+            <th>
+                Description 
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                `LlmChatCompletionMessage`
+            </td>
+            <td>
+                `content`
+            </td>
+            <td>
+                Describes when your model generates text to be delivered to your end user. For example, a completion event is triggered when the model generates a text response containing the address. The final message appers as the value for the attribute `content`.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                `LlmFeedbackMessage`
+            </td>
+            <td>
+                `messages`
+            </td>
+            <td>
+                INSERT_CELL_CONTENT
```

Comment:
I'm sure you won't forget, just leaving this as a bookmark to be filled in later!